pkg/api/annotation_key_constants.go

```diff
@@ -19,6 +19,10 @@ limitations under the License.
 package api
 
 const (
+	// ImagePolicyFailedOpenKey is added to pods created by failing open when the image policy
+	// webhook backend fails.
+	ImagePolicyFailedOpenKey string = "alpha.image-policy.k8s.io/failed-open"
```

Comment:
You need to copy this to the versioned api too (bleh): https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/annotation_key_constants.go